checker/plugins/firejail.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import os
+
+from .scripts import RunScriptPlugin
+
+
+PATH = 'PATH'
+PYTHONPATH = 'PYTHONPATH'
+
+
+class SafeRunScriptPlugin(RunScriptPlugin):
+    """Plugin for running scripts in safe mode."""
+
+    name = "safe_run_script"
+
+    Args = RunScriptPlugin.Args
```

Comment:
Seems it’s better to make custom Args - you need env whitelist, potentially network mode (eg for pip install profile or socket access) , custom allowed dirs -all with default parameters